docs/installation.rst

```diff
@@ -100,12 +100,21 @@ and ``view``. You can also add your own permissions.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 An integer that defines the size of chunks when iterating a QuerySet for data
-exports. Defaults to ``100``. You may be able to save memory usage byd
+exports. Defaults to ``100``. You may be able to save memory usage by
 decreasing it, or speed up exports by increasing it.
 
 Can be overridden on a ``Resource`` class by setting the ``chunk_size`` class
 attribute.
 
+``IMPORT_EXPORT_SKIP_ADMIN_CONFIRM``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If True, no import confirmation page will be presented to the user in the Admin UI.
+The file will be imported in a single step.
+
+The import will occur in a transaction. If the import results in validation errors,
```

Comment:
I think it might be worth being explicit here that this will occur in a transaction even if USE_TRANSACTIONS is false?

Reply:
Thanks - i think this identifies a bug (that `use_transactions` is ignored.  I'll look into it

Reply:
I have corrected this so that it respects the `use_transactions` settings.